exporters/geneva/test/common/generate_metrics.h

```diff
@@ -111,6 +114,96 @@ GenerateSumDataLongMetrics() {
   return data;
 }
 
+// Counter Instrument of long type
+const std::string kUpDownCounterLongInstrumentName =
```

Comment:
nit: seems these string constants are only used once? If so, put them in in the function argument list directly?

Reply:
These strings are used in the unit tests for validation.